Library/Formula/mongo-c.rb

```diff
@@ -14,11 +14,17 @@ class MongoC < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libbson"
-  depends_on "openssl"
+  depends_on "openssl" => :recommended
 
   def install
     # --enable-sasl=no: https://jira.mongodb.org/browse/CDRIVER-447
-    system "./configure", "--prefix=#{prefix}", "--enable-sasl=no"
+    args = ["./configure", "--prefix=#{prefix}", "--enable-sasl=no"]
```

Comment:
Make the args array without `./configure` in it; more standard with other formulae.